website/docs/install/install-with-script.md

```diff
@@ -276,24 +267,24 @@ This will produce the following output:
 
 
 ```
-{"data":{"head_slot":"6944","sync_distance":"3003133","is_syncing":true,"is_optimistic":true}}
+{"data":{"head_slot":"6944","sync_distance":"3003133","is_syncing":true}}
 ```
 
-When you see `"is_syncing":false`, your beacon node is fully synchronized with the beacon chain. When you see `"is_optimistic":false`, your execution node is fully synchronized with the execution-layer blockchain. 
+When you see `"is_syncing":false`, your beacon node is fully synchronized with the beacon chain. 
+
+<!-- When you see `"is_optimistic":false`, your execution node is fully synchronized with the execution-layer blockchain. 
 
 ```
 TODO: is_optimistic is not yet implemented -> https://github.com/prysmaticlabs/prysm/pull/10692 
 
 TODO: `Checking DB database-path=C:\Users\newuser\AppData\Local\Eth2\beaconchaindata` <- how to configure this directory?
 ```
 
+-->
 
-:::tip Congratulations!
-
-You’re now running a <strong>full Ethereum node</strong>. Your full node consists of an <strong>execution node</strong> in Ethereum’s execution layer, and a <strong>beacon node</strong> in Ethereum’s consensus layer.
-
-:::
+Congratulations - you’re now running a <strong>full Ethereum node</strong>. Your full node consists of an <strong>execution node</strong> in Ethereum’s execution layer, and a <strong>beacon node</strong> in Ethereum’s consensus layer.
 
+If you want to support Ethereum Mainnet, ensure that TCP port `8545` is closed. Consider keeping TCP and UDP ports `30303` open to support other execution nodes [<a href='#footnote-8'>8</a>].
```

Comment:
we might want to specify that open and closed refer to the open internet. They should still be open locally on your computer. Perhaps we can say that you can configure a firewall [footnode] to improve your security